oommfc/oommf/oommf.py

```diff
@@ -331,18 +331,28 @@ class DockerOOMMFRunner(OOMMFRunner):
 
         Docker image on DockerHub. Defaults to ``oommf/oommf:20a3``.
 
+    selinux : bool, optional
+
+        If ``True`` use additional ``:z`` flag for the mounted directories. This can be
+        required to get read/write access when using selinux. Use with caution and check
+        the Docker documentation on selinux:
+        https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label.
+        Alternatively, setting ``selinux`` to ``permisive`` has been reported to give
```

Comment:
Setting to `permissive`? Where? User might attempt `selinux='permissive'`

Reply:
Good point. Changing SELinux to `permissive` is completely independent of `ubermag` (on the command line `sudo setenforce Permissive`). Maybe we can change the explanation to:

> Alternatilvely, you can set SELinux to Permissive Mode to get read/write access.

That should avoid confusions. What do you think?

Reply:
Yes, I agree. That would avoid confusion.